### DISCUSSION TITLE
账户被锁定（或显示非法使用被锁定）的处理方法

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1569] [高级账户]
# 简悦 · 高级账户平台

### 地址

> https://simpread.pro/account

### 用法

> 一站式管理你的高级账户，功能包括：**自助解锁** · **查看当前授权设备数量** · **解绑全部设备** 等操作。

### 快速入门

如果你经常会锁定的话，建议看下 [快速掌握](http://ksria.com/simpread/docs/#/%E5%B7%B2%E8%B4%AD%E5%BF%85%E8%AF%BB?id=%e5%bf%ab%e9%80%9f%e6%8e%8c%e6%8f%a1)

### 注意事项

1.  **解绑全部设备** 为 **时限性功能**，因为只要登录过的设备，均可以手动退出来实现解绑。（ 已登录设备，超过 14 天未使用的话，自动解绑）

2. 每周仅能解锁 3次，一般情况下，这个限制周期足够了。

### 常见使用场景

> 以下的 Issues 均来自高级账户真实的使用场景的一些解决方案。

- [非同时在线的情况下，是否可以授权超过三台设备？](https://github.com/Kenshin/simpread/issues/1795)

**Comment by willsyangtx** [comment]
旧设备不用了，我解绑全部设备以后。在自己的新设备上重新输入 licence 绑定，提醒我密钥已被使用怎么办？

**Reply by Kenshin** [reply]
> 旧设备不用了，我解绑全部设备以后。在自己的新设备上重新输入 licence 绑定，提醒我密钥已被使用怎么办？

嗨，授权当前设备为高级账户不是使用 License 方案，如何操作请看 [找回 UID](https://github.com/Kenshin/simpread/discussions/1571) [重新授权](https://github.com/Kenshin/simpread/discussions/1576)

以及 https://github.com/Kenshin/simpread/discussions/categories/高级账户 下你需要的一些操作方式。

**Reply by willsyangtx** [reply]
我知道我自己的 UID，也知道我的 License 的情况下：
我之前授权的设备丢失了，我在自己的新设备上进行了『解除全部设备』操作后进行了如下操作，都没法绑定：
1、登录：提醒我要使用配置文件或第三方登录，但我的配置文件找不到了。丢失了。
2、仅授权当前设备：提醒我您还不是高级用户，请先购买高级用户资格。
3、绑定您的高级用户资格：在这里输入 license ，提醒我密钥已被使用（但我明明解除了全部设备）

以上情况我全部按照指导教程试了一遍，仍然没有用。麻烦帮忙看下。谢谢！

**Reply by Kenshin** [reply]
嗨，请使用 

> 2. 仅授权当前设备：提醒我您还不是高级用户，请先购买高级用户资格。

在使用此方式时，**确保 UID 为注册时的 UID**

当前 UID 查看方式 → 选项页 → 账户 → 用户信息。

***

如果 UID 不一致的话，看我上面的回复内容。

**Reply by willsyangtx** [reply]
系统新分配给我的新的 UID 与 通过 license 找回的 UID 不一致，所以再回到你上面的回复内容。通过以下方式遇到了卡点：
1、配置文件登录（修改为高级账户的 UID ）：因为笔记本丢失，配置文件已经丢失了，找不到
2、授权登录：之前绑定高级帐号以后没有对坚果云 或 Dropbox 进行授权。所以现在无法用这两个途径进行登录

因为笔记本遗失造成了现在的问题，对开发大大多有叨扰。非常抱歉(。・＿・。)ﾉI’m sorry~

**Reply by Kenshin** [reply]
嗨，没事的。

笔记本遗失确实是件悲伤的事情。（摸摸头）

你可以这么做：

首先要找到当时给你的 License ，然后使用 https://github.com/Kenshin/simpread/discussions/1571 找回，找回后会自动帮你授权设备，即可。

如何找到 License 可以 [看看这里](https://simpread.pro/api/#/%E5%A6%82%E4%BD%95%E5%BC%80%E9%80%9A?id=%e5%a6%82%e4%bd%95%e6%89%be%e5%88%b0-license) 。

**Reply by willsyangtx** [reply]
搞定~谢谢大叔！！！非常感谢~~！！！：）

**Reply by Kenshin** [reply]
哈哈，搞定就好，希望一切都快好起来~

PS：第一次被自己的用户叫大叔。 😂

**Reply by willsyangtx** [reply]
😊
